index.html

```diff
@@ -2020,9 +2020,12 @@ <h3>Presentations</h3>
           <dd>
 The <a>verifiable presentation</a> MAY include a <code>proof</code>
 <a>property</a>. If present, the value SHOULD be used to express a securing
-mechanism such as [[?VC-DATA-INTEGRITY]]. A <a>verifiable presentation</a> MAY
-be secured using an external proof such as [[?VC-JWT]]. For details related to
-the use of this property, see Section <a href="#securing-verifiable-credentials"></a>.
+mechanism, such as [[?VC-DATA-INTEGRITY]], that is listed in the
+<a href="https://w3c.github.io/vc-specs-dir/#proof">Proof section</a> of the
+Verifiable Credentials Specifications Directory [[?VC-SPECS]]. A
+<a>verifiable presentation</a> MAY be secured using an external proof such as
```

Comment:
```suggestion
<a>verifiable presentation</a> MAY be secured using an <a>external proof</a> such as
```

The use of a hyperlink here seems consistent with earlier usage.